src/AutoRest.CSharp/Common/Generation/Writers/ModelWriter.cs

```diff
@@ -351,13 +343,13 @@ private FormattableString ConstructBinaryDataDescription(string typeSpecificDesc
 </list>
 </para>";
         }
-        private string GetAbstract(SchemaObjectType schema)
+        private string GetAbstract(ObjectType schema)
         {
             // Limit this change to management plane to avoid data plane affected
             return schema.Declaration.IsAbstract && Configuration.AzureArm ? "abstract " : string.Empty;
         }
```

Comment:
Could we let this to return a boolean (or potentially move this to ObjectType to return a boolean), and in the corresponding place of the writer, we could use
```
writer.AppendRawIf("abstract ", IsAbstract(obj));
```

Reply:
I think we can postpone this clean up for future PRs as this isn't introducing this behavior simply re-using it